spec/models/book_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
```

Comment:
@evan-007 I'm indifferent, I usually use single as well.  It seems like they address it in this issue.  https://github.com/thoughtbot/hound/issues/977